src/Orleans/Serialization/SerializationManager.cs

```diff
@@ -99,9 +99,11 @@ public static object GetUninitializedObjectWithFormatterServices(Type type)
         private static List<IExternalSerializer> externalSerializers;
         private static ConcurrentDictionary<Type, IExternalSerializer> typeToExternalSerializerDictionary;
         private static Dictionary<string, Type> types;
-        private static Dictionary<RuntimeTypeHandle, DeepCopier> copiers;
-        private static Dictionary<RuntimeTypeHandle, Serializer> serializers;
-        private static Dictionary<RuntimeTypeHandle, Deserializer> deserializers;
+        volatile private static Dictionary<RuntimeTypeHandle, DeepCopier> copiers;
+        volatile private static Dictionary<RuntimeTypeHandle, Serializer> serializers;
+        volatile private static Dictionary<RuntimeTypeHandle, Deserializer> deserializers;
```

Comment:
Re: pair of methods, I guess it's fine to have a tuple if they are always assigned as a pair (which does make sense)